### ISSUE TITLE
First setup installation issues

### ISSUE THREAD

**Issue by symdeb** [steinbergmedia/vst3sdk#89]
Following applies to windows 10 and 11

[1] Should be: cmake.exe -G "Visual Studio 17 2022" -SMTG_CREATE_PLUGIN_LINK=0 -A x64 ..\vst3sdk 

[2] Also please add that for  msbuild.exe vstsdk.sln you need to open the Visual Studio developer prompt in ADMNISITRATOR mode .this is from the tart menu "Developer command prompt for Visual Studio 2022" (a bit hard to find)

[3] VS2019 & VS2022: msbuild.exe vstsdk.sln, when building the examples, getting lot of errors in red color such as

```
The system cannot find the file specified.
  Could Not Find C:\Program Files\Common Files\VST3\channelcontext.vst3
  Access is denied.
  [SMTG] Validator finished.
  [SMTG] Delete previous link...
  [SMTG] Creation of the new link...
E:\Tools\vs\vs2022\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(156,5): error MSB3073: The command "setlocal [
E:\Tools\vstsdk\build\public.sdk\samples\vst\channelcontext\channelcontext.vcxproj]
E:\Tools\vs\vs2022\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(156,5): error MSB3073: cd E:\Tools\vstsdk\build\bin [E:\Tools\vstsdk\build\public.sdk\samples\vst\channelcontext\channelcon text.vcxproj]

```
[4] The git repository does not seem to include the .bat for VST2, the zip download from Steinberg website does.

----------------------------------------------------------------------

[3] SOLVED: To avoid the error, run the Developer Tools prompt in Administrator mode
only  errors remain related to the pitchnames project:

"...\VST_SDK\build\vstsdk.sln" (default target) (1) ->
"...\VST_SDK\build\public.sdk\samples\vst\pitchnames\pitchnames.vcxproj.metaproj" (default target) (53) ->
"...\VST_SDK\build\public.sdk\samples\vst\pitchnames\pitchnames.vcxproj" (default target) (54) ->
(ClCompile target) ->

...\VST_SDK\vst3sdk\public.sdk\samples\vst\pitchnames\source\pitchnamesdatabrowsersource.cpp(266,1): fatal error
 C1001: Internal compiler error. [...\VST_SDK\build\public.sdk\samples\vst\pitchnames\pitchnames.vcxproj]
  cl : command line error D8040: error creating or communicating with child process [...\VST_SDK\build\public.sdk\
samples\vst\pitchnames\pitchnames.vcxproj]

**Comment by ygrabit** [comment]
there is an error in the example: use this:
 -DSMTG_CREATE_PLUGIN_LINK=0
(fixed in next update)

**Comment by hockinsk** [comment]
I have developer mode enabled in Windows 11. I run cmake as admin to build the visual studio solution. I run visual studio as admin, I build the example plugins, but each one fails with setlocal error. The .vst3 are created even though build fails, they do not open in any .vst3 host though. I tried building the examples with cmake instead. They successfully build and all pass testing as 'succeeded' but these .vst3 do not load into any .vst3 host or are recognised.
I've spent over a week just trying to build the example plugins and gave up, i'm sure many others will too as this is just too much hassle. I switched to CLAP such was the experience with the vst3sdk. Nothing seems to work as documented, it really needs someone to just follow the documentation on a standard Windows 11 build with Visual Studio and i'm sure it will be obvious where all the documentation is invalid?

I've tried setting predefined/ALL_BUILD and ZERO_CHECK all to 'No' as suggested by others and that didn't make any difference either! What to do?
